st/initscripts

```diff
@@ -27,12 +28,61 @@ set -e
 M0_SRC_DIR=`readlink -f $0`
 M0_SRC_DIR=${M0_SRC_DIR%/*/*}
 
-. $M0_SRC_DIR/utils/functions # report_and_exit
+
+echo $M0_SRC_DIR
+m0t1fs_dir="$M0_SRC_DIR/m0t1fs/linux_kernel/st"
+. $m0t1fs_dir/common.sh
+. $m0t1fs_dir/m0t1fs_common_inc.sh
+. $m0t1fs_dir/m0t1fs_client_inc.sh
+. $m0t1fs_dir/m0t1fs_server_inc.sh
 
 cd $M0_SRC_DIR
 
-sudo ./scripts/install-motr-service -u
-sudo ./scripts/install-motr-service -l
-sudo /usr/bin/time -v ./st/m0t1fs -vv
+N=$1
```

Comment:
'$1', '$2' etc represent the arguments passed to a function/script. Here the calling script is not passing any arguments, these assignments are not valid.

Please use the value of N=4, K=2, S=0, P=12, stride=32

Reply:
Okay I will make the changes.